src/pe_reports/report_generator.py

```diff
@@ -28,39 +21,221 @@
+                cyber_creds,
+                masq_df,
+                insecure_df,
+                vulns_df,
+                assets_df,
+                dark_web_mentions,
+                alerts,
+                top_cves,
+            ) = init(
+                source_html,
+                datestring,
+                org_name,
+                org_uid,
+            )
 
+            # Close pdf
+            file.close()
 
-def main() -> None:
-    """Set up logging and call the generate_reports function."""
+            # Convert to HTML to PDF
+            output_filename = f"{output_directory}/{org_code}-Posture_and_Exposure_Report-{datestring}.pdf"
+            convert_html_to_pdf(source_html, output_filename)
+
+            # Create Crendential Exposure excel file
+            cred_xlsx = f"{output_directory}/{org_code}/compromised_credentials.xlsx"
+            credWriter = pd.ExcelWriter(cred_xlsx, engine="xlsxwriter")
+            hibp_creds.to_excel(credWriter, sheet_name="HIBP_Credentials", index=False)
+            cyber_creds.to_excel(
+                credWriter, sheet_name="Cyber6_Credentials", index=False
+            )
+            credWriter.save()
+
+            # Create Domain Masquerading excel file
+            da_xlsx = f"{output_directory}/{org_code}/domain_alerts.xlsx"
+            domWriter = pd.ExcelWriter(da_xlsx, engine="xlsxwriter")
+            masq_df.to_excel(domWriter, sheet_name="Suspected Domains", index=False)
+            domWriter.save()
+
+            # Create Suspected vulnerability excel file
+            vuln_xlsx = f"{output_directory}/{org_code}/vuln_alerts.xlsx"
+            vulnWriter = pd.ExcelWriter(vuln_xlsx, engine="xlsxwriter")
+            assets_df.to_excel(vulnWriter, sheet_name="Assets", index=False)
+            insecure_df.to_excel(vulnWriter, sheet_name="Insecure", index=False)
+            vulns_df.to_excel(vulnWriter, sheet_name="Verified Vulns", index=False)
+            vulnWriter.save()
+
+            # Create dark web excel file
+            mi_xlsx = f"{output_directory}/{org_code}/mention_incidents.xlsx"
+            miWriter = pd.ExcelWriter(mi_xlsx, engine="xlsxwriter")
+            dark_web_mentions.to_excel(
+                miWriter, sheet_name="Dark Web Mentions", index=False
+            )
+            alerts.to_excel(miWriter, sheet_name="Dark Web Alerts", index=False)
+            top_cves.to_excel(miWriter, sheet_name="Top CVEs", index=False)
+            miWriter.save()
+
+            # grab the pdf
+            pdf = f"{output_directory}/{org_code}-Posture_and_Exposure_Report-{datestring}.pdf"
+
+            (filesize, tooLarge) = embed(
+                output_directory,
+                org_code,
+                datestring,
+                pdf,
+                cred_xlsx,
+                da_xlsx,
+                vuln_xlsx,
+                mi_xlsx,
+            )
+            # Need to make sure Cyhy Mailer doesn't send files that are too large
```

Comment:
Make this comment a bit more direct and add some context.
```suggestion
            # Log a message if the report is too large.  Our current mailer 
            # cannot send files larger than 20MB.
```

Reply:
Addressed [here](https://github.com/cisagov/pe-reports/pull/114/commits/9c39af6967a25d26434abdd25b958c107b77ccbd).